app/src/main/java/com/example/tasky/agenda_details/data/repository/AgendaDetailsRemoteRepositoryImpl.kt

```diff
@@ -113,6 +130,27 @@ class AgendaDetailsRemoteRepositoryImpl(
         }
     }
 
+    override suspend fun deleteTask(taskId: String): Result<Unit, DataError.Network> {
+        return try {
+            val response = api.deleteTask(taskId = taskId)
+
+            if (response.isSuccessful) {
+                val responseBody = response.body()
+                if (responseBody != null) {
+                    Result.Success(Unit)
+                } else {
+                    Log.e("Error", "API call failed with code ${response.code()}")
+                    Result.Error(DataError.Network.SERVER_ERROR)
+                }
```

Comment:
Did you test this? I'm not sure this behaves as expected 🤔 
Just because an API doesn't respond with a body, doesn't mean the request failed. If the status code starts with a 2, the request was successful (which is already checked by `response.isSuccessful`)

Reply:
you are right ! will fix this